Nef_3/include/CGAL/Nef_3/shell_to_nef_3.h

```diff
@@ -62,9 +62,10 @@ class Shell_to_nef_3
 
  private:
   SNC_structure& S;
+  bool with_marks;
```

Comment:
what do you think about renaming it to `preserve_marks`?

Reply:
Sure I can do that.